### ISSUE TITLE
Error "STREAM.once is not a function" in 0.80.0 when pulling files

### ISSUE THREAD

**Issue by vlasscontreras** [mkloubert/vscode-deploy-reloaded#81]
### Description

On the latest version, files cannot be pulled anymore. The pull function used to work before the update (but to be honest I can't confirm that 0.80.0 introduced this issue).

### Actual behavior

When you try to pull a file from a target configured via SFTP (haven't tested with other types of target), the output says:

```
[08:55:56] 🚚 Pulling file [...] from  [...] ...🔥: 'STREAM.once is not a function' [186 ms]
```

And the file is not pulled.

### Steps to reproduce

- Get the version 0.80.0
- Set up a target (see example config below)
- Make sure the file exists on remote
- Pull the file (whether via button, left click or command palette)
- See the output tab

#### Example config

```json
{
    	"deploy.reloaded": {
		"targets": [
			{
				"type": "sftp",
				"name": "My Server",
				"description": "My Server",

				"host": "###.###.###.###",
				"dir": "/nasa/classified/aliens/",

				"user": "username",
				"privateKeyPassphrase": "pass",
				"privateKey": "/Users/user/.ssh/id_rsa",
			},
		],
		"packages": [
			{
				"name": "Pull",
				"button": {
					"text": "Pull File",
					"tooltip": "Click here to pull the current file",
					"command": "extension.deploy.reloaded.pullFile"
				}
			},
			{
				"name": "Push",
				"button": {
					"text": "Push File",
					"tooltip": "Click here to push the current file",
					"command": "extension.deploy.reloaded.deployFile"
				}
			}
		],
	}
}
```

#### Logs

```
INFO extension.deploy.reloaded.loadplugins - [02/Jul/2018:13:54:12 +0000] "Loaded 19 plugins:
- app
- azureblob
- batch
- compiler
- dropbox
- each
- ftp
- list
- local
- mail
- map
- prompt
- s3bucket
- script
- sftp
- slack
- switch
- test
- zip
"
WARN workspaces.workspace.reloadpackagebuttons() - [02/Jul/2018:13:54:12 +0000] "Button of package 'Pull' will use the existing command 'extension.deploy.reloaded.pullFile'."
WARN workspaces.workspace.reloadpackagebuttons() - [02/Jul/2018:13:54:12 +0000] "Button of package 'Push' will use the existing command 'extension.deploy.reloaded.deployFile'."
INFO extension.deploy.reloaded.loadplugins - [02/Jul/2018:14:55:46 +0000] "Loaded 19 plugins:
- app
- azureblob
- batch
- compiler
- dropbox
- each
- ftp
- list
- local
- mail
- map
- prompt
- s3bucket
- script
- sftp
- slack
- switch
- test
- zip
"
WARN workspaces.workspace.reloadpackagebuttons() - [02/Jul/2018:14:55:46 +0000] "Button of package 'Pull' will use the existing command 'extension.deploy.reloaded.pullFile'."
WARN workspaces.workspace.reloadpackagebuttons() - [02/Jul/2018:14:55:46 +0000] "Button of package 'Push' will use the existing command 'extension.deploy.reloaded.deployFile'."

```

### Your environment

- Operating system: Microsoft Windows 10 / MacOS 10.13.5
- Visual Studio Code version: 1.24.1
- Extension version: 0.80.0

**Comment by mkloubert** [comment]
@vlasscontreras 

**The bug should be fixed since version 0.80.1.**

[ssh2-sftp-client](https://www.npmjs.com/package/ssh2-sftp-client) seems to have changed the download API in the meantime ... very strange.

**Comment by vlasscontreras** [comment]
@mkloubert 
Just tested the new version, it works perfectly! Thank you!
